### PR TITLE
Add policy-reporter

### PR DESCRIPTION
**Pull request by stone-z**
### Checklist

- [x] Update changelog in CHANGELOG.md.
- [ ] Make sure `values.yaml` and `values.schema.json` are valid.

### CODE REVIEW

**Comment by stone-z** [comment]
This all deploys fine but UI only has partial data for some reason. Worked OK with upstream 1.8.5

File: helm/kyverno/values.yaml [hunk 1]
```diff
@@ -179,3 +179,18 @@ serviceMonitor:
 # 3) Let Helm generate a self signed cert, by setting createSelfSignedCert true
 # If letting Kyverno create its own CA or providing your own, make createSelfSignedCert is false
 createSelfSignedCert: false
+
+# Additional options defined in charts/policy-reporter/values.yaml. Upstream docs: https://github.com/kyverno/policy-reporter
+policy-reporter:
```
Comment (stone-z) [inline helm/kyverno/values.yaml:184]: Overrides for subchart(s)

**Comment by stone-z** [comment]
UI still has partial data, I suspect it is a bug introduced by a recent change upstream. Will tackle that separately

**Comment by stone-z** [comment]
Tested on `gauss` and will leave it running there if anyone wants to see

File: helm/kyverno/charts/policy-reporter/charts/kyvernoPlugin/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+apiVersion: v2
+name: kyvernoPlugin
+description: Policy Reporter Kyverno Plugin
+
+type: application
+version: 0.6.2
+appVersion: 0.3.2
```
Comment (MarcelMue) [inline helm/kyverno/charts/policy-reporter/charts/kyvernoPlugin/Chart.yaml:7]: Newline missing

File: helm/kyverno/values.yaml (hunk 1 above)
Comment (MarcelMue) [reply helm/kyverno/values.yaml:184]: Should we add more resources / limits here for the subcharts?

File: helm/kyverno/charts/policy-reporter/charts/kyvernoPlugin/Chart.yaml (hunk 2 above)
Comment (stone-z) [reply helm/kyverno/charts/policy-reporter/charts/kyvernoPlugin/Chart.yaml:7]: I'd rather not override upstream here but will add a note to fix upstream

File: helm/kyverno/values.yaml (hunk 1 above)
Comment (stone-z) [reply helm/kyverno/values.yaml:184]: Added some requests and limits based on current usage on `gauss`

**Review by MarcelMue — APPROVED** [review]
Let's goooo

**Pull request merged**